monkey/tests/unit_tests/infection_monkey/exploit/zerologon_utils/test_dc_utils.py

```diff
@@ -21,8 +21,8 @@ def stub_queryIPForName(*args, **kwargs):
 
 
 @pytest.mark.slow
-def test_get_dc_details_multiple_netbios_names(host, monkeypatch):
-    from infection_monkey.exploit.zerologon_utils.vuln_assessment import get_dc_details
+def test_dc_utils__get_dc_details_multiple_netbios_names(host, monkeypatch):
```

Comment:
Why have the file name in the test name?
```suggestion
def test_get_dc_details_multiple_netbios_names(host, monkeypatch):
```